config/dbconfig.txt

```diff
@@ -3,7 +3,7 @@
 ## administration, and the in game library.
 
 ## Should SQL be enabled? Uncomment to enable
-#SQL_ENABLED
+SQL_ENABLED
```

Comment:
note to self to remove this when testing is done

Reply:
testings done, remove this when you want this merged because this is nice to have